db/schema.rb

```diff
@@ -11,15 +11,15 @@
 # It's strongly recommended that you check this file into your version control system.
 
 ActiveRecord::Schema[7.0].define(version: 2022_08_15_143028) do
-  create_table "alerts", id: :integer, charset: "utf8mb4", force: :cascade do |t|
```

Comment:
These changes to schema.rb should not be included in your PR.

Reply:
okay